hack/make/dyngccgo

```diff
@@ -15,7 +16,7 @@ fi
 
 (
 	export IAMSTATIC="false"
-	export EXTLDFLAGS_STATIC_DOCKER=''
+	export EXTLDFLAGS_STATIC_DOCKER='-static-libgcc -static-libgo -lnetgo'
```

Comment:
@estesp Thank you for your comments.

To the first question, I enabled `netgo` also in `dyngccgo` since it eliminates dependency to runtime libraries of a GCCGO trunk, which we cannot expect a specific version is installed. We can fix this when GO 1.4 becomes available with a release version of GCCGO.

To the second question, yes, you are correct. I'll remove the redundant `-lnetgo` here.